include/libnuraft/raft_server.hxx

```diff
@@ -220,6 +220,11 @@ public:
      * Append and replicate the given logs.
      * Only leader will accept this operation.
      *
+     * Users need to take care of the 2 points
+     *    1. Limit the logs size in one batch
+     *    2. Limit the speed of appending entries,
+     *       user can get the gap of log store and state machine by NuRaft API.
```

Comment:
I don't think this additional statement is necessary, as it is too trivial and use-case-specific. 

For example, think about `pwrite` API, it sounds like
* Users should limit the size of buffer passed to `pwrite`, or it may cause IO failure.
* Users should limit the speed of disk writes, or it may cause IO failure.

"How to handle failure" is totally up to users. Someone will be ok with the above cases with their special log store and state machine implementations.

Reply:
ok